### PR TITLE
Update TestsRequestHelperTrait.php

### PR DESCRIPTION
**Pull request by Kyslik**
Add correct PHPDoc so IDEs know what is going on.

### CODE REVIEW

File: Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php [hunk 1]
```diff
@@ -74,7 +74,7 @@
      * @param array $data
      * @param array $headers
      *
-     * @return  mixed
+     * @return  mixed|\Illuminate\Foundation\Testing\TestResponse
```
Comment (johannesschobel) [inline Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php]: the `mixed` should be removed, right?
Furthermore, can you add the `@throws UndefinedMethodException` as well?

Comment (Kyslik) [reply Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php]: Yes, you are right, will do tomorrow. :) thanks!

Comment (johannesschobel) [reply Traits/TestsTraits/PhpUnit/TestsRequestHelperTrait.php]: no problem - you're welcome..

**Comment by Mahmoudz** [comment]
Cool, I'll merge when mixed is removed.

**Comment by Kyslik** [comment]
Done :)

**Pull request merged**